addon/templates/components/polaris-resource-list/filter-control/filter-value-selector.hbs

```diff
@@ -0,0 +1,41 @@
+{{#if (eq filter.type FilterType.Select)}}
+  {{#polaris-stack vertical=true as |stack|}}
+    {{#if showOperatorOptions}}
+      {{#stack.item}}
+        {{render-content operatorOptionsMarkup}}
+      {{/stack.item}}
+    {{/if}}
+    {{polaris-select
```

Comment:
nit: some spacing before this maybe

Reply:
Normally I would, but decided to match the React source here 🤷‍♂️